docs/reference/commandline/volume_create.md

```diff
@@ -47,11 +47,25 @@ Some volume drivers may take options to customize the volume creation. Use the `
 These options are passed directly to the volume driver. Options for
 different volume drivers may do different things (or nothing at all).
 
-*Note*: The built-in `local` volume driver does not currently accept any options.
+*Note*: The built-in `local` driver on Windows does not support any options.
+
+*Note*: The built-in `local` driver on Linux accepts options similar to the `mount`
```

Comment:
perhaps "similar to the Linux `mount` command" (so that people don't start looking for `docker mount`

Can we add a link to that command somewhere? Or wouldn't that make sense?